tests/components/airthings/test_config_flow.py

```diff
@@ -10,27 +10,26 @@
 from homeassistant.const import CONF_ID
 from homeassistant.core import HomeAssistant
 from homeassistant.data_entry_flow import FlowResultType
-from homeassistant.helpers.service_info.dhcp import DhcpServiceInfo
 
-from tests.common import MockConfigEntry
+from tests.common import MockConfigEntry, MockDhcpServiceInfo
 
 TEST_DATA = {
     CONF_ID: "client_id",
     CONF_SECRET: "secret",
 }
 
 DHCP_SERVICE_INFO = [
-    DhcpServiceInfo(
+    MockDhcpServiceInfo(
         hostname="airthings-view",
         ip="192.168.1.100",
         macaddress="00:00:00:00:00:00",
     ),
-    DhcpServiceInfo(
+    MockDhcpServiceInfo(
         hostname="airthings-hub",
         ip="192.168.1.101",
         macaddress="D0:14:11:90:00:00",
     ),
-    DhcpServiceInfo(
+    MockDhcpServiceInfo(
         hostname="airthings-hub",
         ip="192.168.1.102",
         macaddress="70:B3:D5:2A:00:00",
```

Comment:
The MAC here was assumed to be upper-case with colons, when it's really lower-case without colons